### DISCUSSION TITLE
Falla en validación de peso negativa

### DISCUSSION THREAD

**Discussion by VirBruno** [frlp-utn-ingsoft/catedra#125] [Q&A] [answered]
Buenas tardes, 

Para validar que el peso de la mascota NO sea negativo, coloqué el siguiente código en models.py:
def validate_fields(data, required_fields):
    errors = {}

    for key, value in required_fields.items():
        field_value = data.get(key, "")
        print(field_value)
        if field_value == "":
            errors[key] = f"Por favor ingrese un {value}"
        elif key == 'email' and field_value.count("@") == 0:
            errors["email"] = "Por favor ingrese un email valido"
        elif key == 'weight' and int(field_value) < 0:
            print(field_value)
            errors["weight"] = "El peso de la mascota no puede ser negativo"

    return errors

Al correr esto, veo que entra a la validación del if, pero en pantalla no se muestra el error. 
![Uploading image.png…]()
![Uploading image.png…]()

**Comment by RodrigoJacznik** [comment]
Hola,
pasame una captura del código del formulario por favor.

**Reply by VirBruno** [reply]
![image](https://github.com/frlp-utn-ingsoft/catedra/assets/87098993/b48191ee-60ad-49fc-834d-4b6b3d59da9e)

**Reply by VirBruno** [reply]
Vi el error de tipeo y lo corregí, pero sigue igual

**Reply by RodrigoJacznik** [reply, marked as answer]
agregale min=0 al input de weight y se deberia ver

**Reply by VirBruno** [reply]
Ahí anduvo, gracias profe!

**Comment by VirBruno** [comment]
![app](https://github.com/frlp-utn-ingsoft/catedra/assets/87098993/b551614a-2175-4540-a82a-9d0a555313f0)
![Captura por consola](https://github.com/frlp-utn-ingsoft/catedra/assets/87098993/2806ff9f-4b6f-4cc0-8cd8-ccbfd67e8d97)
